nextflow_schema.json

```diff
@@ -75,6 +75,62 @@
         }
       }
     },
+    "mags_upload": {
+      "title": "MAGs uploader arguments",
+      "type": "object",
+      "description": "genome_uploader supported options for automatic mags upload",
+      "default": "",
+      "properties": {
+        "webin_account": {
+          "type": "string",
+          "default": "",
+          "description": "Webin account id in format (Webin-XXX)"
+        },
+        "webin_password": {
+          "type": "string",
+          "default": "",
+          "description": "password for Webin account"
+        },
+        "centre_name": {
+          "type": "string",
+          "default": "",
+          "description": "name of the centre generating and uploading genomes"
+        },
+        "upload_tpa": {
+          "type": "boolean",
+          "default": true,
+          "description": "if uploading TPA (Third PArty) generated genomes"
+        },
+        "upload_mags": {
+          "type": "boolean",
+          "default": true,
+          "description": "Upload MAGs (Metagenome-Assembled Genomes)"
+        },
+        "upload_bins": {
+          "type": "boolean",
+          "default": false,
+          "description": "Upload bins"
+        }
+      },
+      "oneOf": [
+        {
+          "properties": {
+            "webin_account": { "type": "string" },
+            "webin_password": { "type": "string" },
+            "centre_name": { "type": "string" },
+            "upload_tpa": { "type": "boolean" },
+            "upload_mags": { "const": true },
+            "upload_bins": { "const": false }
```

Comment:
I'm not sure how this works.. does this mean that the user need to provide "at least" one this values?